docs: update Docker instructions (#901)
The-OpenROAD-Project/OpenROAD-flow: docs/user/BuildWithDocker.md

make
exit
```

## Build Docker Image for Different OS

The following instructions build the docker image with a parameterized OS
in two stages. These are for CI and developers that wish to use an OS other

> than CentOS 7, regular users should use the steps from previous sections.

Reviewer: ```suggestion
than CentOS 7; regular users should use the steps from previous sections.
```

> Dev stage for installing all the dependencies and packages that are needed
> to run OpenROAD and OpenROAD Flow Scripts. The build stage actually builds
> all of the binaries needed to run flow (i.e., `openroad` and `yosys`).

Reviewer: ```suggestion
The dev stage installs all dependencies and packages required to run OpenROAD
and OpenROAD Flow Scripts. The build stage generates all binaries needed to run
the flow (i.e., `openroad` and `yosys`).
```

make
exit
```

## Enable GUI support

To use the GUI feature you will need to start the docker with the following command:

```

> docker run --rm -it \

Reviewer 2: It is good to have this documented.  You could add a flow/Makefile target like 'make mount' to do this more easily.

Author: Merging docs now to deploy new readthedocs page. Will add the command later.
